datafusion/proto/Cargo.toml

```diff
@@ -56,4 +56,4 @@ serde_json = { workspace = true, optional = true }
 [dev-dependencies]
 doc-comment = { workspace = true }
 strum = { version = "0.26.1", features = ["derive"] }
-tokio = "1.18"
+tokio = { version = "1.18", features = ["rt-multi-thread"] }
```

Comment:
is this change needed?

Reply:
```error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
  --> datafusion/proto/examples/logical_plan_serde.rs:22:1
   |
22 | #[tokio::main]
   | ^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)```
   
The above error is hit when `rt-multi-thread` is absent.